bionic/gcs.py

```diff
@@ -101,22 +101,35 @@ def blobs_matching_url_prefix(self, url_prefix):
         obj_prefix = self._validated_object_name_from_url(url_prefix)
         return self._bucket.list_blobs(prefix=obj_prefix)
 
-    def gsutil_cp(self, src_url, dst_url):
-        args = [
-            "gsutil",
-            "-q",  # Don't log anything but errors.
-            "-m",  # Transfer files in paralle.
-            "cp",
-            "-r",  # Recursively sync sub-directories.
-            src_url,
-            dst_url,
-        ]
-        logger.debug("Running command: %s" % " ".join(args))
-        subprocess.check_call(args)
-        logger.debug("Finished running gsutil")
+    @staticmethod
```

Comment:
If it's not too much work, I think creating a `Gsutil` type class would certainly make this better. I think that class would be equivalent to your `FakeGCS` class. But I agree that it might not be worth it if it's too much, given our inclination to move to fsspec.

Reply:
Yeah, dependency injection may be neater, but global mocks are much easier to set up.